nni/algorithms/hpo/gridsearch_tuner.py

```diff
@@ -88,7 +88,7 @@ class GridSearchTuner(Tuner):
         config.tuner.name = 'GridSearch'
     """
 
-    def __init__(self):
+    def __init__(self, optimize_mode=None):
```

Comment:
a little strange but it's ok for me, maybe let user config this on web portal is better (Top max 10 or Top min 10, default visualization is `optimize_mode` if it has)...